PROJECT

```diff
@@ -0,0 +1,20 @@
+domain: kiegroup.org
```

Comment:
This is the group which is appended to our CRDs right so right now, so right now we have CRD as `kogitoinfras.app.kiegroup.org` of we change it becomes `kogitoinfras.app.kie.org`, I think we can think about changing it later but not in this same PR, personally I am not sure if changing the group in a middle version is a good idea but this can be discussed :)

Reply:
fair enough :)